routes/projects.js

```diff
@@ -0,0 +1,82 @@
+const express = require("express");
+
+const projects = require("../data/helpers/projectModel.js");
+const router = express.Router();
+
+router.get("/", async (req, res) => {
+  try {
+    const theProject = await projects.get();
+    res.status(200).json(theProject);
+  } catch {
+    res.status(500).json({ error: "There was a problem getting the project" });
+  }
+});
+
+router.get("/:id", async (req, res) => {
+  try {
+    const theProject = await projects.get(req.params.id);
+    res.status(200).json(theProject);
+  } catch {
+    res.status(500).json({ error: "There was a problem getting the project" });
+  }
+});
+
+router.get("/:id/actions", async (req, res) => {
+  try {
+    const theProject = await projects.get(req.params.id);
+    console.log(theProject);
```

Comment:
You don't need to `.get()` the projects first. If the project doesn't exist then `.getProjectActions()` shouldn't return any data